examples/pose-apriltag/readme.md

```diff
@@ -0,0 +1,100 @@
+# rs-pose-apriltag Sample
+
+In order to run this example, a T265 is required.
+
+## Overview
+
+[Apriltags](https://april.eecs.umich.edu/software/apriltag) are fiducial markers that have been designed to be easily detected in images (other fiducial markers include QR codes and ARTags). Once detected, these markers can be used for a variety of applications.
+
+This sample demonstrates how to detect the pose of a Apriltag of a known size that is visible in a T265 fisheye frame relative to the camera. Apriltags are detected on every 6th frame (at 5Hz). The sample uses `36h11` tags, though it can be easily modified to work with any tag supported by the Apriltag library. 
+
+A sample `36h11` tag image (tag size is 0.144m on a letter size paper) pdf can be downloaded [here](./tag36_11_00000_out180mm_in144mm.pdf), and other tag images can be found [here](https://github.com/AprilRobotics/apriltag-imgs). 
+
+## Dependency
+
+This sample requires the [Apriltag library 3.1.1](https://github.com/AprilRobotics/apriltag/tree/3.1.1)  installed in the default location suggested by the library or a path specified in cmake variable `CMAKE_PREFIX_PATH`.
```

Comment:
In order for the demo to be built into distribution images (both Win and Linux) all the dependencies should be embedded into the project's Cmake ( see [Libusb](https://github.com/IntelRealSense/librealsense/blob/master/third-party/libusb/CMakeLists.txt) dependency, for instance).
So for now it will be available via the manual build only.

Reply:
@ev-mp  I think we are okay via manual build at this moment.